### ISSUE TITLE
Wear OS 4, Passive Data, Heart Rate collecting not working in background

### ISSUE THREAD

**Issue by amoledwatchfaces** [android/health-samples#101]
Background Heart Rate collection is not working on Wear OS 4. Tried with own app (PassiveListenerCallback) but also with PassiveData sample (PassiveDataService.kt)

**Comment by jluebbe** [comment]
@amoledwatchfaces Did you get this to work on Wear OS 4? I think I have the same issue.

**Comment by amoledwatchfaces** [comment]
Hi, it's not really an issue. Wear OS is killing app in the background after some time of inactivity. This applies for both PassiveListenerCallback / PassiveListenerService. I though that PassiveListenerService will work like WearableListenerService so it wont be killed by system but unfortunately, it's not the case.

**Comment by breanatate** [comment]
@jluebbe Are you also requesting the [body sensors background](https://developer.android.com/health-and-fitness/guides/health-services/background-body-sensors) permission?

**Comment by cancan101** [comment]
I solved this problem by requesting the `BODY_SENSORS_BACKGROUND` permission. Without doing so, once the app was closed, I no longer received updates (either data points or user activity).

I was a bit confused by the linked docs, however. It states that the app should "Check if `BODY_SENSORS_BACKGROUND` is granted. If not, request the permission" but then down below it states "the runtime permission dialog doesn't include the "Allow all the time" option. Instead, users must enable all-the-time background sensor access from system settings".

What is the point of checking (and requesting) the `BODY_SENSORS_BACKGROUND` if the user needs to enable it manually?
